src/libraries/Microsoft.Extensions.HostFactoryResolver/src/HostFactoryResolver.cs

```diff
@@ -31,6 +40,35 @@ internal sealed class HostFactoryResolver
             return ResolveFactory<THostBuilder>(assembly, CreateHostBuilder);
         }
 
+        public static Func<string[], object>? ResolveHostFactory(Assembly assembly, TimeSpan? waitTimeout = null, bool stopApplication = true, Action<object>? configureHostBuilder = null)
```

Comment:
I don't see a test that uses `configureHostBuilder`. Do we need this parameter?

Reply:
Yes, we will need it for https://github.com/dotnet/aspnetcore/blob/main/src/Mvc/Mvc.Testing/src/WebApplicationFactory.cs

Reply:
Ok, then it would be great if we had a test ensuring it doesn't break.

Reply:
2 things:
1. As of right now, ASP.NET Core will need to use it so it kinda *has* to work or tests will fail regardless 😄.
2. I agree with you and will add a test.